docs/install-audit-log-filter.md

```diff
@@ -0,0 +1,42 @@
+# Install the Audit Log Filter
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The `plugin_dir` system variable defines the plugin library location. If needed, at server startup, set the `plugin_dir` variable.
+
+When upgrading a MySQL installation, plugins are not automatically upgraded. You may need to manually load certain functions or reinstall the plugin after the MySQL upgrade.
```

Comment:
Please remove "manually load certain functions or" here. There may be a need to reload the plugin but we do not install functions manually.